packages/core/src/decorators/attachment/Attachment.ts

```diff
@@ -45,6 +47,16 @@ export class AttachmentData {
     }
   }
 
+  public getDataAsJson() {
+    if (this.base64 !== null && typeof this.base64 === 'string') {
+      return JsonEncoder.fromBase64(this.base64)
+    } else if (this.json !== null) {
+      return this.json
+    } else {
+      throw new AriesFrameworkError('No attachment data available.')
```

Comment:
Maybve we can be a bit more explicit in the error message, as it could be that data is available, just not in the base64 or json fields

`No attachment data found in 'json' or 'base64' data fields.`